src/main/java/com/gmail/llmdlio/townyflight/config/TownyFlightConfig.java

```diff
@@ -86,6 +86,8 @@ private void loadConfig(){
     	addDefault("options.disable_During_Wartime","true");
     	addComment("options.disable_Combat_Prevention","# If set to false, TownyFlight will not prevent combat of flying people.");
     	addDefault("options.disable_Combat_Prevention","false");
+	addComment("options.show_Permission_After_No_Permission_Message""# If set to false, the language.noPermission message will not display the permission node.");

```

Comment:
missing a comma between "options.show_Permission_After_No_Permission_Message" and "# If set to false, the language.noPermission message will not display the permission node."

Reply:
Apologies, I'm new to github, didnt know it would put both checkins in the pull request, will fix it right away